<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateConfigWindow</class>
 <widget class="QDialog" name="CreateConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create config</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="volumeComboBox"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="ConfigSettingsWidget" name="configSettingsWidget" native="true"/>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QGroupBox" name="templateGroupBox">
     <property name="title">
      <string>Use template</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="templateComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createPushButton">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="configNameLineEdit">
     <property name="placeholderText">
      <string>Config name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigSettingsWidget</class>
   <extends>QWidget</extends>
   <header>widgets.ConfigSettingsWidget.ConfigSettingsWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateConfigWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>templateGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>configSettingsWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
